app/uk/gov/hmrc/alcoholdutyaccount/connectors/helpers/Headers.scala

```diff
@@ -0,0 +1,52 @@
+/*
+ * Copyright 2024 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.alcoholdutyaccount.connectors.helpers
+
+import play.api.http.HeaderNames
+import uk.gov.hmrc.alcoholdutyaccount.config.AppConfig
+import uk.gov.hmrc.alcoholdutyaccount.utils.DateTimeHelper
+
+import java.time.{Clock, Instant}
+import java.util.Base64
+import javax.inject.Inject
+
+class Headers @Inject() (randomUUIDGenerator: RandomUUIDGenerator, appConfig: AppConfig, clock: Clock) {
+  private val correlationIdHeader: String       = "correlationId"
```

Comment:
These are the data specifically needed for HIP, so please rename to something like hip headers. Can use injection as well to help future migration to other systems

Reply:
Done